02week/pigLatin.js

```diff
@@ -3,49 +3,57 @@
 const assert = require('assert');
 const readline = require('readline');
 const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 
 
 function pigLatin(word) {
-
-  // Your code here
-
+    word = word.toLowerCase().trim();
+    if (word.charAt(0).match(/[aeiouy]/)) {
+        return word.concat('yay');
+    }
+    return (
+        word
+            .slice(word.search(/[aeiouy]/))
```

Comment:
For these, I'd leave the daisy chained .'s on the same line as word, since if a line starts with a dot, it's assumed it's off of the line above. I think this is best practice. Not 100% sure, but I've seen this done before.